content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "माइक्रोप्रोफाइल, कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करके कॉन्फिगरेशन को बाहरी बनाना"
+content_type: ट्यूटोरियल
+weight: 10
+---
+
+<!-- overview -->
+
+इस ट्यूटोरियल में आप सीखेंगे कि अपने माइक्रोसर्विस के कॉन्फ़िगरेशन को कैसे और क्यों बाहरी बनाना है। 
+विशेष रूप से, आप सीखेंगे कि पर्यावरण चर सेट करने के लिए कुबेरनेट्स कॉन्फिगमैप्स और सीक्रेट्स का 
+उपयोग कैसे करें और फिर माइक्रोप्रोफाइल कॉन्फिग का उपयोग करके उनका उपभोग करें। 
+
+
+## {{% heading "prerequisites" %}}
+
+### कुबेरनेट्स कॉन्फ़िगमैप्स और रहस्य बनाना
+
+कुबेरनेट्स में डॉकर कंटेनर के लिए पर्यावरण चर सेट करने के कई तरीके हैं, जिनमें शामिल हैं: डॉकरफाइल, 
+कुबेरनेट्स.वाईएमएल, कुबेरनेट्स कॉन्फिगमैप्स और कुबेरनेट्स सीक्रेट्स। ट्यूटोरियल में, आप सीखेंगे कि अपने 
+पर्यावरण चर सेट करने के लिए कुबेरनेट्स कॉन्फिगमैप्स और कुबेरनेट्स सीक्रेट्स का उपयोग कैसे करें, जिनके मान 
+आपके माइक्रोसर्विसेज में इंजेक्ट किए जाएंगे। कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करने का एक लाभ यह है कि 
+उन्हें कई कंटेनरों में फिर से उपयोग किया जा सकता है, जिसमें विभिन्न कंटेनरों के लिए अलग-अलग पर्यावरण चर 
+को सौंपा जाना भी शामिल है।
+
+कॉन्फिगमैप्स एपीआई ऑब्जेक्ट हैं जो गैर-गोपनीय कुंजी-मूल्य जोड़े को संग्रहीत करते हैं। 
```

Comment:
कुंजी-मूल्य this doesnt  easily understanable  instead you can write `key-value` also `पेयर्स`